src/api/axiosInstance.ts

```diff
@@ -21,7 +21,7 @@ import axiosWrapperCreateRequestInstance from './axiosWrapper';
  * @param {callback} resolve Callback to be stored and used in case of a retry after a fail
  * @param {number} timeout Timeout in milliseconds for the request
  */
-export const defaultCreateRequestInstance = (resolve, timeout) => {
+export const defaultCreateRequestInstance = (resolve?: null, timeout?: number) => {
```

Comment:
`resolve` is not `null`.

Reply:
`defaultCreateRequestInstance` only uses it as an argument to `axiosWrapperCreateRequestInstance`, defined [here](https://github.com/HathorNetwork/hathor-wallet-lib/blob/09a75f7f5700ae50a7ed8b0a03162a8c185b1d80/src/api/axiosWrapper.ts#L28) as `null`.